models/fund.py

```diff
@@ -62,5 +53,7 @@ def get_service_name():
         if short_name:
             fund_data = FundMethods.get_fund(fund_short_name=short_name)
         else:
-            fund_data = FundMethods.get_fund(fund_id=Config.DEFAULT_FUND_ID)
+            fund_data = FundMethods.get_fund(
+                fund_id=Config.DEFAULT_FUND_ID, use_short_name=False
```

Comment:
get_fund doesn't take an arg called fund_id?
Do we have a unit test for this method?

Reply:
ahh you're right, i recon we can remove that whole else block entirely 👍🏾

Reply:
we do have a test but it looks outdated, let me fix this to!